src/commands/scheduled/facilitate-incident-review.ts

```diff
@@ -0,0 +1,88 @@
+import { flags } from "@oclif/command"
+import Command from "../../base"
+import { Opsgenie } from "../../utils/opsgenie"
+import { convertEmailsToSlackMentions } from "../../utils/slack"
+
+export default class FacilitateIncidentReview extends Command {
+  static description =
+    "Choose a random on-call participant to facilitate the Incident Review"
+  static urls: { [key: string]: string } = {
+    incidentReviewSchedule:
+      "https://www.notion.so/artsy/Incident-Reviews-725052225efc49e78532b13e166ba3c7",
+    onCallEngineeringSchedule:
+      "https://artsy.app.opsgenie.com/teams/dashboard/ee381004-a72e-42ef-a733-b350d6693c6c/main",
+  }
+
+  static flags = {
+    ...Command.flags,
+    date: flags.string({
+      description: "target date in ISO format",
+      default: wednesday11AM(),
+    }),
+    schedule: flags.string({
+      description: "schedule name",
+      default: "Engineering On Call",
+    }),
+    facilitatorEmail: flags.string({
+      description: "facilitator email",
+      default: "",
```

Comment:
**question**: Should this have a explicit default if it's an empty string and expected to be falsey? Be default would this allow an undefined default value?

Reply:
Nope, the default isn't needed and it should not allow an `undefined` default. Removed in https://github.com/artsy/cli/pull/302/commits/7de43f6c9f1637a4b9e5c6abd0726ff63a01b9ea and instead use `||` when initializing `email`.

I hope this improves the implementation. Thanks for asking about this.